Tests/StreamChatUITests/AssertSnapshot.swift

```diff
@@ -43,7 +43,7 @@ func AssertSnapshot(
     variantsToTest.forEach { variant in
         assertSnapshot(
             matching: viewController,
-            as: .image(size: screenSize, traits: variant.traits),
+            as: .image(precision: 0.99, size: screenSize, traits: variant.traits),
```

Comment:
any reason to changing precision?

Reply:
The tests fail between M1 vs Intel for tiny differences. It makes sense to lower the precision 👍